aws/cloudformation/update_certs

```diff
@@ -2,7 +2,7 @@
 #
 # Creates/updates LetsEncrypt SSL certificates stored in S3 using acmesmith.
 # Ref: https://github.com/sorah/acmesmith
-# Usage: update_certs [domain]
+# Usage: update_certs COMMON_NAME [SAN_1] [SAN_2] [...]
```

Comment:
Could you include a note in the comments to explain that SAN stands for Subject Alternative Name